test/unit/service/jobRoleServiceTest.js

```diff
@@ -0,0 +1,38 @@
+var axios = require('axios');
+var MockAdapter = require('axios-mock-adapter');
+var chai = require('chai');  
+const expect = chai.expect;
+const JobRoleService = require('../../../dist/src/services/jobRoleService');
+const job = {
```

Comment:
Should this have a type?

Reply:
I didn't know how to assign it a type

Reply:
It looks to me like this is a job role object to help us out in tests? In typescript, the type comes after the variable name like this

```suggestion
const job: JobRole = {
```